tests/test_client.py

```diff
@@ -1391,3 +1396,71 @@ def test_read_metadata_remediation_message(client: GGClient):
     assert client.remediation_messages.pre_commit == messages["pre_commit"]
     assert client.remediation_messages.pre_push == messages["pre_push"]
     assert client.remediation_messages.pre_receive == messages["pre_receive"]
+
+
+LIST_MEMBERS_RESPONSE = [
+    {
+        "id": 3251,
+        "name": "Owl",
+        "email": "john.smith@example.org",
+        "role": "owner",
+        "access_level": "owner",
+        "active": True,
+        "created_at": "2022-04-20T11:07:24.000Z",
+        "last_login": "2022-04-20T11:07:24.000Z",
+    },
+    {
+        "id": 3252,
+        "name": "Owl",
+        "email": "john.smith@example.org",
+        "role": "owner",
+        "access_level": "owner",
+        "active": True,
+        "created_at": "2022-04-20T11:07:24.000Z",
+        "last_login": "2022-04-20T11:07:24.000Z",
+    },
+]
+
+
+@my_vcr.use_cassette("test_list_members.yaml", ignore_localhost=False)
+def test_list_members(client: GGClient):
+    """
+    GIVEN a client
+    WHEN calling /members endpoint
+    THEN it returns a paginated list of members
+    """
+
+    result = client.list_members()
+
+    assert isinstance(result, CursorPaginatedResponse), result.content
```

Comment:
It's better to use `result` as the error message: because `Detail` does not have a `content` field.